templates/pages/setup/general/base_form.html.twig

```diff
@@ -54,3 +54,27 @@
         </div>
     </form>
 {% endif %}
+
+<script>
+    $(() => {
+        // Handle anchor in URL if present since this content is loaded via AJAX
+        if (window.location.hash && $(window.location.hash).length > 0) {
```

Comment:
Is the code in `src/Ajax.php` (l386) doesn't work already ? It seems like a doublon for me

Reply:
No. It seems like the auto-scroll and animation are only triggered when you switch tabs and doesn't work for the initial tab load. If that is resolved, then I can probably remove the code for it in this PR.

Reply:
pushed a commit for that, could you check if it's ok for you